config/_default/hugo.toml

```diff
@@ -18,7 +18,7 @@ disableLanguages = ["de", "nl"]
 defaultContentLanguageInSubdir = false
 
 # Footer
-footer = '<small>© Copyright 2024, <a href="https://www.sap.com/corporate/en/legal/copyright.html">SAP SE </a> and Open Component Model Contributors | <a href="https://www.sap.com/about/legal/trademark.html">Trademark </a> | <a href="/privacy-policy/">Privacy Policy</a> | <a href="https://www.sap.com/corporate/en/legal/terms-of-use.html">Terms of Use</a> | <a href="https://www.sap.com/corporate/en/legal/impressum.html">Legal Disclosure</a></small>'
+footer = '<small>© Copyright 2024, <a href="https://www.sap.com/corporate/en/legal/copyright.html">SAP SE </a> and Open Component Model Contributors | <a href="https://www.sap.com/about/legal/trademark.html">Trademark </a> | <a href="https://www.sap.com/about/legal/privacy.html">Privacy Policy</a> | <a href="https://www.sap.com/corporate/en/legal/terms-of-use.html">Terms of Use</a> | <a href="https://www.sap.com/corporate/en/legal/impressum.html">Legal Disclosure</a></small>'

```

Comment:
the following changes are already introduced by https://github.com/open-component-model/ocm-website/pull/322. Probably a rebase should solve the duplicated changes